src/applications/personalization/profile360/components/PaymentInformationEditModalError.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+
+const ACCOUNT_FLAGGED_FOR_FRAUD = 'cnp.payment.flashes.on.record.message';
+const INVALID_ROUTING_NUMBER = 'payment.accountRoutingNumber.invalidCheckSum';
+
+function FlaggedAccount() {
+  return (
+    <>
+      <p>
+        We’re sorry. You can’t change your direct deposit information right now
+        because we’ve locked your account. We do this to protect your bank
+        account information and prevent fraud when we think there may be a
+        security issue.
+      </p>
+      <p>
+        If you have any questions, please call us at 800-827-1000 (TTY:
+        800-829-4833), and select 5. We’re here Monday through Friday, 8:00 a.m.
+        to 9:00 p.m. ET.
+      </p>
+    </>
+  );
+}
+
+function InvalidRoutingNumber() {
+  return (
+    <p>
+      We couldn’t find a bank linked to this routing number. Please check your
+      bank’s 9-digit routing number and enter it again.
+    </p>
+  );
+}
+
+function hasError(errors, errorKey) {
+  return errors.some(err => err.key === errorKey);
+}
+
+export default function PaymentInformationEditModalError({ responseError }) {
+  if (!responseError) return null;
+
+  const { errors = [] } = responseError.error;
+  let content = null;
+
+  if (hasError(errors, ACCOUNT_FLAGGED_FOR_FRAUD)) {
```

Comment:
could inline these conditions in the JSX with ternary to make it more declarative. 

`{hasError(errors, invalid_routing_number) &&  <InvalidRoutingNumber />}`

Reply:
I would like to do this but I'm trying to limit how many server-side errors are shown to just one and I also want a fallback error message. I think I'd have to nest ternaries to do that.